docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -321,7 +321,7 @@ However, if your model contains more than one input, Model Optimizer is able to
 
 #### Q49. What does the message "Data type is unsupported" mean? <a name="question-49"></a>
 
-**A** : Model Optimizer cannot convert the model to the specified data type. Currently, FP16 and FP32 are supported. Make sure you specify the data type with the `--data_type` flag. The available values are: FP16, FP32, half, float.
+**A** : Model Optimizer cannot convert the model to the specified data type. Currently, FP16 and FP32 are supported. For compressing weights in the model to FP16 specify `--compress_to_fp16` flag.
```

Comment:
This message is not correct. We support more types, not only FP16 and FP32. Please find the root for this message and reformulate it accordingly. Is it still related to `--compress_to_fp16`? I don't understand where this message (in Q49) may appear. I also think that many other messages in this FAQ lost their meaning and they require a major refresh.

Reply:
This message occurs in TF extractor if type is not in found in list of known types: https://github.com/openvinotoolkit/openvino/blob/3c8578fd5b17e0cd1bbb5a26401a691d1cc5ad12/tools/mo/openvino/tools/mo/front/tf/common.py#L10
I added these types in FAQ, as supported.
Looks like it is not related to fp16 compression.